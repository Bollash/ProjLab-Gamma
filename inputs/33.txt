Create Space
Add Asteroid Iron
Add Settler 0
Mod Counter 1
Countdown
Status All