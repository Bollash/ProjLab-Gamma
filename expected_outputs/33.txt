Space létrehozva
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
Sikeres volt a telepeshozzáadás
A napszél visszaszámláló az adott értékre állítódott.
A számláló elérte a nullát és napszél lett. A számláló visszaállt a kiindulási állapotba.
Space
0
10
10
True
1
0
Asteroid
0
3
3
3
Iron
0
0
